data_steward/tools/generate_unioned_ehr_dataset.sh

```diff
@@ -104,6 +136,36 @@ bq rm -f ${unioned_ehr_dataset}._mapping_ipmc_nu_procedure_occurrence
 bq rm -f ${unioned_ehr_dataset}._mapping_ipmc_nu_specimen
 bq rm -f ${unioned_ehr_dataset}._mapping_ipmc_nu_visit_occurrence
 
+# Run cleaning rules on unioned_ehr_dataset
+# create an intermediary table to apply cleaning rules on
+bq mk --dataset --description "intermediary dataset to apply cleaning rules on ${unioned_ehr_dataset_backup}" --label "phase:staging" --label "release_tag:${dataset_release_tag}" --label "de_identified:false" ${app_id}:${unioned_ehr_dataset_staging}
+
+"${TOOLS_DIR}/table_copy.sh" --source_app_id ${app_id} --target_app_id ${app_id} --source_dataset ${unioned_ehr_dataset_backup} --target_dataset ${unioned_ehr_dataset_staging}
+
+export UNIONED_DATASET_ID="${unioned_ehr_dataset_staging}"
+export BIGQUERY_DATASET_ID="${unioned_ehr_dataset_staging}"
+data_stage='unioned'
+
+# Remove de-activated participants
+if [ ${deactivation} == 'true' ];
+then
+  python "${CLEANER_DIR}/cleaning_rules/remove_ehr_data_past_deactivation_date.py" --project-id ${app_id} --pids-project-id ${pids_project_id} --pids-dataset-id ${pids_dataset_id} --pids-table ${pids_table}
+fi
+
+# run cleaning_rules on a dataset
+python "${CLEANER_DIR}/clean_cdr.py" --data_stage ${data_stage} -s 2>&1 | tee unioned_cleaning_log_"${unioned_ehr_dataset_staging}".txt
+
+# Create a snapshot dataset with the result
+python "${TOOLS_DIR}/snapshot_by_query.py" -p "${app_id}" -d "${unioned_ehr_dataset_staging}" -n "${unioned_ehr_dataset}"
```

Comment:
The long form of the command line arguments would be nice here.  
```suggestion
python "${TOOLS_DIR}/snapshot_by_query.py" --project_id "${app_id}" --dataset_id "${unioned_ehr_dataset_staging}" --snapshot_dataset_id "${unioned_ehr_dataset}"
```